tests/src/test/scala/cats/tests/SyntaxTests.scala

```diff
@@ -137,6 +140,11 @@ object SyntaxTests extends AllInstances with AllSyntax {
 
     val fga = mock[F[G[A]]]
     val gunit: G[F[A]] = fga.sequence
+
+    val ft = mock[(A, B, C) => G[Z]]
+
+    tfabc traverseN ft
+    (fa, fb, fc) traverseN ft
```

Comment:
Either add a `()` at the end of the method  or make it a statement by adding a `val blah = (fa, fb, fc) traverseN ft` which is a ()